tasks/app/views/tasks/index.html.erb

```diff
@@ -1,36 +1,63 @@
-<!-- <header>
-  <div class="outer-box">
-    <div class="inner-box">
-      <h1>THINGS TO DO</h1>
-    </div>
-  </div>
-</header>
-<hr> -->
 <div class="list">
   <div class ="title">
     List
   </div>
   <% gray = true %>
   <% num = 1 %>
+  <!--Creates task list with alternating gray and white boxes, and strikes through completed tasks  -->
   <% @tasks.each do |task| %>
     <% if task.complete.nil? %>
       <% if gray %>
         <div class="gray-box">
           <span><%= num %>    </span><a href="/tasks/<%= task.id %>"><%= task.name.upcase %></a>
+          <%= button_to "delete", "/tasks/#{task.id}", method: :delete, data: { :confirm => "Are you sure?" }, class: "delete-button" %>
         </div>
         <% gray = false %>
         <% num += 1 %>
+      <!--else the box is white -->
```

Comment:
I think you can use a conditional for the class that is added to the div as opposed to using a conditional for this entire section of code. Especially since you have to duplicate a lot of HTML here which is no good.